### ISSUE TITLE
Can this pretrained model generate pseudo-labels for unseen classes

### ISSUE THREAD

**Issue by liaowang0125** [LUSSeg/PASS#2]
Since I see the pretrain process is not related to the label，is it possible to generate pseudo-labels for unseen classes using the provided pretrained models and centroids? Or can I directly use your pretrained model to generate class centers on our own data from step 2.2,and then generate pseudo-labels

**Comment by gasvn** [comment]
I think it depends on your data. If your data is similar to the ImageNet-S dataset, I think you can use our pretrained model to generate pseudo-labels or class centers. Otherwise, you may need to train/finetune the model on your own data.

**Comment by liaowang0125** [comment]
ok

**Comment by liaowang0125** [comment]
I used your pretrain model and centroids to generate pseudo labels for the training set(**from Imagenet1k**), but it doesn't look very accurate, what is the possible reason？here is my command`CUDA_VISIBLE_DEVICES=1 python3 inference_pixel_attention.py --pretrained pass919_pixelatt.pth.tar --dump_path dump -c 919 --mode val --centroid pass919_centroids.npy -t 0.5`
![image](https://user-images.githubusercontent.com/20236379/200497533-ad5554b3-6512-4458-bfcc-b175d8b41b02.png)
![image](https://user-images.githubusercontent.com/20236379/200497569-9a422415-b374-4a2e-82e9-249262ff8022.png)

**Comment by gasvn** [comment]
Well, maybe this is one of the failure cases, as the performance on LUSS is not good enough for now. Can you please provide the name of this image so we can check if it is a failure case or caused by other reasons?
